polyfill/index.d.ts

```diff
@@ -351,8 +351,8 @@ export namespace Temporal {
     negated(): Temporal.Duration;
     abs(): Temporal.Duration;
     with(durationLike: DurationLike): Temporal.Duration;
-    add(other: Temporal.Duration | DurationLike | string, options?: DurationOptions): Temporal.Duration;
-    subtract(other: Temporal.Duration | DurationLike | string, options?: DurationOptions): Temporal.Duration;
+    add(other: Temporal.Duration | DurationLike | string, options?: DurationRoundOptions): Temporal.Duration;
```

Comment:
I assume that the options are defined here but not in the docs because you're waiting for #856 to document the new options?  OK if so.

Reply:
That's right.